projects/igniteui-angular/src/lib/grids/grid-base.component.ts

```diff
@@ -4907,7 +4921,7 @@ export abstract class IgxGridBaseComponent extends DisplayDensityBase implements
     }
 
     openRowOverlay(id) {
-        this.configureRowEditingOverlay(id);
+        this.configureRowEditingOverlay(id, this.rowList.length <= 2);
```

Comment:
What is `2`?

Reply:
I've added a comment about that.

Reply:
Well, according to this comment, there should be something like `private MIN_ROW_EDITING_COUNT_THRESHOLD = 2`

The comment should also go above the constant, rather than above where it's used.